ee/codegen/src/__test__/generate-code-fixtures/two-generic-nodes-with-same-base-coalesce.ts

```diff
@@ -0,0 +1,242 @@
+export default {
+  workflow_raw_data: {
+    nodes: [
+      {
+        id: "entrypoint-node",
+        type: "ENTRYPOINT",
+        data: {
+          label: "Entrypoint",
+          source_handle_id: "entrypoint-source",
+        },
+        inputs: [],
+      },
+      {
+        id: "first-generic-node",
+        type: "GENERIC",
+        label: "First Generic Node",
+        data: {
+          source_handle_id: "first-generic-source",
+          target_handle_id: "first-generic-target",
+        },
+        inputs: [],
+        display_data: {
+          position: {
+            x: 500,
+            y: 200,
+          },
+          z_index: 1,
+          comment: null,
+          width: 330.0,
+          height: 96.0,
+          icon: null,
+          color: null,
+        },
+        base: {
+          name: "SharedBaseNode",
+          module: ["testing", "nodes", "shared_base_node"],
+        },
+        definition: {
+          name: "FirstGenericNode",
+          module: ["testing", "nodes", "first_generic_node"],
+        },
+        trigger: {
+          id: "first-generic-target",
+          merge_behavior: "AWAIT_ANY",
+        },
+        ports: [
+          {
+            id: "first-generic-source",
+            name: "default",
+            type: "DEFAULT",
+          },
+        ],
+        adornments: null,
+        outputs: [
+          {
+            id: "first-output-id",
+            name: "value",
+            type: "STRING",
+            value: undefined,
+          },
+        ],
+        attributes: [],
+      },
+      {
+        id: "second-generic-node",
+        type: "GENERIC",
+        label: "Second Generic Node",
+        data: {
+          source_handle_id: "second-generic-source",
+          target_handle_id: "second-generic-target",
+        },
+        inputs: [],
+        display_data: {
+          position: {
+            x: 500,
+            y: 400,
+          },
+          z_index: 1,
+          comment: null,
+          width: 330.0,
+          height: 96.0,
+          icon: null,
+          color: null,
+        },
+        base: {
+          name: "SharedBaseNode",
+          module: ["testing", "nodes", "shared_base_node"],
+        },
+        definition: {
+          name: "SecondGenericNode",
```

Comment:
It's important for this test that this field matches the first node's definition.name, so in this case, `FirstGenericNode`